projects/cdk/services/meta-color.service.ts

```diff
@@ -0,0 +1,44 @@
+import type {OnDestroy} from '@angular/core';
+import {inject, Injectable} from '@angular/core';
+import {WA_WINDOW} from '@ng-web-apis/common';
+import {Subject} from 'rxjs';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class TuiMetaColorService implements OnDestroy {
+    private readonly document = inject(WA_WINDOW).document;
+    private readonly value = new Subject<string>();
+
+    protected sub = this.value.subscribe({
+        next: (value) => this.metaThemeColor.setAttribute('content', value),
```

Comment:
We also have a CSS variable used in Navigation — `--tui-theme-color`. Let's also set it on `documentElement`

Reply:
fixed